model/pokemon.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const mongoose = require('mongoose');
+const createError = require('http-errors');
+const Trainer = require('./trainer.js');
+
+const Schema = mongoose.Schema;
+
+const pokemonSchema = new Schema({
+  name: {type: String, required:true},
+  type: {type:String, required:true},
+  pokedexNum: {type:Number, required:true},
+  moves: {type:Array, required:false},
```

Comment:
You don't have to put ``required: false`` in this field. Just not including a ``required`` property suffices. :)